src/Model/ModelManagerInterface.php

```diff
@@ -293,6 +295,9 @@ public function modelReverseTransform($class, array $array = []);
      */
     public function modelTransform($class, $instance);
 
+    // NEXT_MAJOR: Uncomment this.
+//    public function supportQuery(object $query): bool;
```

Comment:
```suggestion
//    public function supportsQuery(object $query): bool;
```

Reply:
I didn't know if I should do this.

For instance, we have a method `createQuery`, not `createsQuery`

Reply:
I think that is because "create" is an imperative verb. In this case, "supports" is a question.
This method should even be named "isQuerySupported".
See this example from [`symfony/security-core`](https://github.com/symfony/symfony/blob/80e5a7fbd80ef04fced05bb3d4bcb8816fa897be/src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php#L70).

Reply:
Thanks for the explanation.

Reply:
Done !